pkg/infrastructure/services/prometheus.go

```diff
@@ -25,7 +27,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
-	"net/http"
```

Comment:
I believe we are missing a `go fmt` alignment here. I'll take a look later. You don't need to worry about it now.